src/redux/store.ts

```diff
@@ -1,5 +1,8 @@
-import { createStore, applyMiddleware } from "@reduxjs/toolkit";
-import thunk from "redux-thunk";
+import { configureStore } from "@reduxjs/toolkit";
 import rootReducer from "./reducer";
 
-export default createStore(rootReducer, applyMiddleware(thunk));
+const store = configureStore({ reducer: rootReducer });
```

Comment:
configureStore is redux toolkit's wrapper around createStore which includes thunk
https://redux-toolkit.js.org/usage/usage-guide#simplifying-store-setup-with-configurestore